src/activity_actor.cpp

```diff
@@ -1449,6 +1449,12 @@ bool read_activity_actor::npc_read( npc &learner )
 
 void read_activity_actor::finish( player_activity &act, Character &who )
 {
+    if( !book ) {
+        debugmsg( "%s lost book item_location while reading", who.disp_name() );
```

Comment:
```suggestion
```

I don't think that needs an on-screen debug message.

Reply:
It's for the case of when the player loses the book for whatever reason. A bug where the NPC itself losing the book should already be quite rare.

Reply:
This can happen during normal game play so a normal message might be better.